ttnn/ttnn/operations/conv2d.py

```diff
@@ -33,6 +33,78 @@ def get_conv_output_dim(input, window, stride=1, pad=0, dilation=1):
     return (input + (2 * pad) - dilation * (window - 1) - 1) // stride + 1
 
 
+def prepare_conv_weights(
+    *,
+    weight_tensor,
+    input_memory_config,
+    weights_format,
+    in_channels,
+    out_channels,
+    batch_size,
+    input_height,
+    input_width,
+    kernel_size,
+    stride,
+    padding,
+    dilation,
+    groups,
+    device,
+    conv_config=None,
+):
+    return ttnn._ttnn.operations.conv2d.prepare_conv_weights(
+        weight_tensor=weight_tensor,
+        input_memory_config=input_memory_config,
+        weights_format=weights_format,
+        in_channels=in_channels,
+        out_channels=out_channels,
+        batch_size=batch_size,
+        input_height=input_height,
+        input_width=input_width,
+        kernel_size=list(kernel_size),
+        stride=list(stride),
+        padding=list(padding),
+        dilation=list(dilation),
+        groups=groups,
+        device=device,
+        conv_config=conv_config,
+    )
+
+
+def prepare_conv_bias(
```

Comment:
Shouldn't new ops use the new op framework?

If you use that, then bind_registered_operation should remove the need for this extra code.

Reply:
It's more of a host-only helper at the moment. If the metal team wants me to do it I can go for it I suppose